certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -157,6 +157,49 @@ def test_certonly(context: IntegrationTestsContext) -> None:
     assert_cert_count_for_lineage(context.config_dir, 'newname', 1)
 
 
+@pytest.skip("Fails")
+def test_certonly_ecdsa_account_flag(context: IntegrationTestsContext) -> None:
+    context.certbot([
+        'certonly',
+        '--register-unsafely-without-email',
+        '--cert-name', 'newname',
+        '--ecdsa-account-key',
+        '-d', context.get_domain('newname'),
+    ])
+
+    privkey = join(context.config_dir, 'live', 'newname', 'privkey.pem')
```

Comment:
This is the path to the cert key. Don't we want the path to the account key?

Reply:
Yes, I feel a bit dumb. Noticed last night also, when I tried to make some progress. Probably been staring at it too long :)

I think introducing a new assert utility in `assertions.py` to check for the JWK-type of the account key (reading it from JSON) would be a good idea.

Reply:
No worries! Having a fresh set of eyes helps a lot sometimes.